lib/requester/util.js

```diff
@@ -116,29 +118,37 @@ module.exports = {
      *
      * @todo: Move this to the SDK.
      * @param request
+     * @param options
      */
-    getRequestBody: function (request) {
-        var options,
+    getRequestBody: function (request, options) {
+        var reqBody,
```

Comment:
can we rename this variable to `computedBody` and move it at the end of `var` definition?